ftw/news/tests/test_news_portlets.py

```diff
@@ -545,7 +544,7 @@ def test_news_portlet_no_footer(self, browser):
 
         self.assertEqual(
             [],
-            browser.css('dl.portlet.newsTemplate li.portletFooter'),
+            browser.css('news-portlet footer'),
```

Comment:
CSS expression might be invalid: there should not be a tag `<news-portlet>` :wink: